docs/timeout-activities-and-workflows.md

```diff
@@ -0,0 +1,119 @@
+---
+id: timeout-activities-and-workflows
+title: How to timeout Activities and Workflows
+sidebar_label: Timeout Activities and Workflows
+---
+
+Temporal does not impose limits on the duration of Activities or Workflows by default, it is designed to enable these to run idefinitely if desired. Instead, timeout options are available to set limits on the time allowed for certain actions to occur.
+
+## Activity timeouts
+
+Activity timeouts provide additional control over the way your Workflows execute Activities.
+
+### Timeout policy options
```

Comment:
We need to explain which of these timeouts are required. As far as I remember only one timeout is required and it should be either StartToClose or ScheduleToClose. 

Defaults are:
If ScheduleToClose is specified:

- ScheduleToStart defaults to ScheduleToClose
- StartToClose defaults to ScheduleToClose

If only StartToClose is specified both ScheduleToStart and ScheduleToClose default to WorkflowRun timeout.